include/queue.php

```diff
@@ -29,6 +29,25 @@ function queue_run(&$argv, &$argc){
 		// Handling the pubsubhubbub requests
 		proc_run(PRIORITY_HIGH,'include/pubsubpublish.php');
 
+		$interval = ((get_config('system','delivery_interval') === false) ? 2 : intval(get_config('system','delivery_interval')));
```

Comment:
You have again removed code in this PR. This part was removed because Friendica now only uses the worker. Please remove this part.